src/main/scala/org/apache/mesos/chronos/scheduler/config/StatsPublisherConfiguration.scala

```diff
@@ -0,0 +1,14 @@
+package org.apache.mesos.chronos.scheduler.config
+
+import org.rogach.scallop.ScallopConf
+
+trait StatsPublisherConfiguration extends ScallopConf{
+  lazy val webhookUrl = opt[String]("stats_webhook_url",
+    descr = "Url to which tasks statistics are pushed",
+    default = None)
+
+  lazy val webhookPort = opt[Int]("stats_webhook_port",
+    descr = "Port where the stats are pushed",
+    default = Some(8080))
```

Comment:
Small, but the default specified is the default marathon port I believe. Given marathon is often used to run Chronos, might be a good idea to choose a different port.